docs/_includes/Running-Spark-on-Alluxio/fault-tolerant-mode-with-zookeeper-bash.md

```diff
@@ -1,7 +1,4 @@
 ```bash
-export SPARK_JAVA_OPTS="
-  -Dalluxio.zookeeper.address=zookeeperHost1:2181,zookeeperHost2:2181
-  -Dalluxio.zookeeper.enabled=true
-  $SPARK_JAVA_OPTS
-"
+spark.driver.extraJavaOptions -Dalluxio.zookeeper.address=zookeeperHost1:2181,zookeeperHost2:2181 -Dalluxio.zookeeper.enabled=true
```

Comment:
no, we don't need quotes because this is parsed by java. If they are quoted, the quotes will be treated as chars as well.  
I didn't try zookeeper. i only tried this and it worked:
spark.executor.extraJavaOptions -Dalluxio.user.file.writetype.default=CACHE_THROUGH -Dalluxio.user.block.size.bytes.default=8MB 
spark.driver.extraJavaOptions -Dalluxio.user.file.writetype.default=CACHE_THROUGH -Dalluxio.user.block.size.bytes.default=8MB